03app_inriamuseum_showroom/03app_inriamuseum_showroom.c

```diff
@@ -64,6 +64,8 @@ typedef struct {
 
 app_dbg_t app_dbg;
 
+uint8_t f_hp = 0;
```

Comment:
- do not define variables outside of the app_vars and app_dbg struct please
- do not initialize at delcaration, but explicitly in the init fuction. If you need 0, then the `memset` sets it to the right value
- inclear what this variable does, give it a better name